### PR TITLE
Bug 1771335: Revert buildah v1.11.4 bump

### PR DESCRIPTION
**Pull request by adambkaplan**
This reverts commit 7486d29c59f5c64d8ae28c264c3a390b9f0315cb, reversing
changes made to b51329eca9192de5fbfd00d49193bcc4fa8ea4c2.

### CODE REVIEW

File: hack/update-deps.sh [hunk 1]
```diff
@@ -18,7 +18,27 @@ for pkg in "$GOPATH/src/k8s.io/kubernetes/staging/src/k8s.io/"*; do
   fi
 done
 
-export GO111MODULE=on
-go mod tidy
-go mod vendor
-go mod verify
```
Comment (adambkaplan) [inline hack/update-deps.sh]: @gabemontero @nalind we can restore this in a separate PR.

File: go.mod [hunk 2]
```diff
@@ -69,6 +69,7 @@ require (
 )
 
 replace (
+	github.com/containers/image/v4 => github.com/containers/image/v4 v4.0.0-20191004175620-9da78d1f50d2
```
Comment (adambkaplan) [inline go.mod]: note that this may not be necessary, but was removed in original PR

Comment (gabemontero) [reply go.mod]: confirmed commit 9da78d1f50d2 was the level for my containers/image fix for handling mounted secrets container `.dockercfg` files

**Comment by adambkaplan** [comment]
/hold

Reverting to 1.11.3 did not address the issue

**Comment by adambkaplan** [comment]
/close

Revert did not fix the bug.

**Pull request closed without merging**